synthtool/gcp/templates/java_library/.github/workflows/ci.yaml

```diff
@@ -55,17 +55,10 @@ jobs:
         java-version: 8
     - run: java -version
     - name: Install artifacts to local Maven repository
-      run: |
-        source .kokoro/common.sh
-        retry_with_backoff 3 10 \
-            mvn install -B -V \
-                -Dmaven.test.skip -DskipTests=true \
-                -Dclirr.skip=true \
-                -Denforcer.skip=true \
-                -Dmaven.javadoc.skip=true \
-                -Dgcloud.download.skip=true
+      run: .kokoro/build.sh
```

Comment:
The build.sh has the following mvn command.

```
retry_with_backoff 3 10 \
  mvn install -B -V \
    -DskipTests=true \
    -Dclirr.skip=true \
    -Denforcer.skip=true \
    -Dmaven.javadoc.skip=true \
    -Dgcloud.download.skip=true \
    -T 1C
```

Without `JOB_TYPE` environment variable, the script just finishes the installation of the artifacts to the local Maven repository. Suitable behavior for Linkage Monitor.